src/crate/theme/rtd/sidebartoc.py

```diff
@@ -0,0 +1,141 @@
+        border = " border-top" if border_top else ""
+        if project == project_name:
+            parts.append(f'<li class="current{border}">')
+            parts.append(f'<a class="current-active" href="{master_path}">{display_name}</a>')
+            if include_toctree:
+                parts.append(get_toctree())
+            parts.append('</li>')
+        else:
+            if only_if_current_project:
+                return
+            parts.append(f'<li class="navleft-item{border}"><a href="{url_if_not_current}">{display_name}</a></li>')
+
+
+    # Special project used standalone
+    if project == 'SQL 99':
+        current_class = ' class="current"' if pagename == master_doc else ''
+        parts.append(f'<li{current_class}>')
+        parts.append(f'<a class="current-active" href="{master_path}">SQL-99 Complete, Really</a>')
+        parts.append(get_toctree(maxdepth=2))
+        parts.append('</li>')
+        return ''.join(parts)
+
+
+    # Start CrateDB docs TOC with a Search box
+    parts.append('<li>')
+    parts.append('<div class="search-link">')
+    parts.append('<div id="docsearch" style="min-height: 36px; margin-bottom: 20px;"></div>')
+    parts.append('</div>')
+    parts.append('</li>')
+
+    # Add Overview and top level entries defined in the Guide's toctree
+    if project == 'CrateDB: Guide':
+        if pagename == 'index':
+            parts.append('<li class="current">')
+            parts.append(f'<a class="current-active" href="{master_path}">Overview</a>')
+            parts.append('</li>')
+        else:
+            parts.append('<li class="navleft-item">')
+            parts.append(f'<a href="{master_path}">Overview</a>')
+            parts.append('</li>')
+        parts.append(get_toctree())
+    else:
+        parts.append('<li class="current"><a class="current-active" href="#">Overview</a></li>')
+
+    # Add individual projects
+    _add_project_nav_item('CrateDB Cloud', 'CrateDB Cloud', '/docs/cloud/')
+    _add_project_nav_item('CrateDB: Reference', 'Reference Manual', '/docs/crate/reference/')
+
+    # Start new section with a border
+    _add_project_nav_item('CrateDB: Admin UI', 'Admin UI', '/docs/crate/admin-ui/', border_top=True)
+    _add_project_nav_item('CrateDB: Crash CLI', 'CrateDB CLI', '/docs/crate/crash/')
+    _add_project_nav_item('CrateDB Cloud: Croud CLI', 'Cloud CLI', '/docs/cloud/cli/')
+
+    # Add all Driver projects
+    parts.append('<li class="navleft-item">')
+    parts.append('<a href="/docs/guide/connect/drivers.html">Database Drivers</a>')
+    parts.append('</li>')
+
+    _DRIVER_CONFIGS = [
+        ('CrateDB JDBC', 'JDBC', '/docs/jdbc/'),
+        ('CrateDB DBAL', 'PHP DBAL', '/docs/dbal/'),
+        ('CrateDB PDO', 'PHP PDO', '/docs/pdo/'),
+        ('CrateDB Python', 'Python', '/docs/python/'),
+        ('SQLAlchemy Dialect', 'SQLAlchemy', '/docs/sqlalchemy-cratedb/'),
+        ('CrateDB Npgsql', '.NET Npgsql', '/docs/npgsql/')
```

Comment:
CrateDB Npgsql is really legacy and very much outdated, let's remove it. The canonical examples exclusively use vanilla Npgsql.

- https://github.com/crate/cratedb-examples/tree/main/by-language/csharp-npgsql
- https://github.com/crate/cratedb-examples/tree/main/by-language/csharp-efcore
```suggestion
```